website/OpenEXRFileLayout.rst

```diff
@@ -991,177 +991,858 @@ pixels. The image has two channels: G, of type ``HALF``, and Z, of type
+indicate how those basic objects form compound objects such as attributes 
+or the line offset table.
+
+Download the :download:`sample.exr <downloads/sample.exr>`.
+
+.. table::
+  :width: 50%
+
+  +-----------+------------+-------------+
+  |byte       |value       |description  |
+  +===========+============+=============+
+  |76         |20000630    |magic number |
+  +-----------+            |             |
+  |2f         |            |             |
+  +-----------+            |             |
+  |31         |            |             |
+  +-----------+            |             |
+  |01         |            |             |
+  +-----------+------------+-------------+
+  |02         |2           |version,     |
+  +-----------+            |flags        |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  | Start of header                      |
+  +-----------+------------+-------------+
+  |63         |c           |attribute    |
+  +-----------+------------+name:        |
+  |68         |h           |channels     |
+  +-----------+------------+             |
+  |61         |a           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |65         |e           |             |
+  +-----------+------------+             |
+  |6c         |l           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |63         |c           |attribute    |
+  +-----------+------------+type:        |
+  |68         |h           |chlist       |
+  +-----------+------------+             |
+  |6c         |l           |             |
+  +-----------+------------+             |
+  |69         |i           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |74         |t           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |25         |37          |attribute    |
+  +-----------+            |size         |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  | Attribute value: list of channels    |
```

Comment:
This seems out of place here. I like the `Start of header` above as an overall indication of the coming section of the file, but this subsection is just the value of the current attribute. This one is unusual since the value is not a simple scalar value.